src/models/tiles/log/log-tile-base-event.ts

```diff
@@ -15,7 +15,9 @@ export function isTileBaseEvent(params: Record<string, any>): params is ITileBas
 function processTileBaseEventParams(params: ITileBaseLogEvent) {
   const { document, tileId, ...others } = params;
   const sectionId = document?.content?.getSectionIdForTile(tileId);
-  return { document, tileId, sectionId, ...others };
+  const tile = document?.content?.getTile(tileId);
+  const tileTitle = tile?.title ?? "<no title>";
```

Comment:
This mechanism for getting the title isn't going to work for tiles that for one reason or another don't store their title in the usual place, notably table tiles. The canonical way of handling that is to use the `getTitle()` method of the `ITileApi`. Unfortunately for this purpose, the tile api is managed by the `CanvasComponent` and made available via React context, neither of which is accessible from the logging code. The simplest thing to do in the short run may be to just hard-code a special case for any tiles that don't follow the usual pattern, which are usually legacy tiles of some sort. In the interest of not proliferating such workarounds, I suggest adding a `getTileTitle()` method to the `DocumentContentModel` (perhaps after `getTileType()`) and then handle any special cases there. It's not elegant but it seems the simplest way to not got bogged down into more abstraction and refactoring. (The `<no title>` default for logging purposes should perhaps be a shared constant somewhere, perhaps even in `logger.ts`).

Reply:
@kswenson Thanks. That makes a lot of sense. One thing still isn't clear to me, though, sorry.

For all the tile types specified by the story, I seem to get a valid value from `tile.title`, including table tiles. I see, however, that although the value in a table tile's header cell initially matches the tile title, the header cell value can be changed, but changing that value does not change the tile's title value. The tile title seems to always remain the initial value.

Are you saying we need to treat the value in the header cell as the table tile's actual title for the logs? And so for table tiles we'll have to make sure to pass that value instead of `tile.title`?

Reply:
I spoke with @tealefristoe about this elsewhere. He answered my question above. He also suggested looking into possibly updating the table tile's title whenever the table cell value is changed. I'm going to take a look into that and see if it would be relatively easy to do.